src/xdocs/config_naming.xml

```diff
@@ -176,7 +176,7 @@
         </source>
         <p>Example:</p>
         <div class="wrapper">
-          <pre>
+          <source>
```

Comment:
Source tags don't need the div wrapper outside it. Please remove it for them all and fix the indentation for source.
See https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_modifier.xml#L98

Reply:
Done.